README.md

```diff
@@ -219,7 +219,8 @@ The actual capture framerate may be printed to the console:
 scrcpy --print-fps
 ```
 
-It may also be enabled or disabled at any time with <kbd>MOD</kbd>+<kbd>i</kbd>.
+It may also be enabled or disabled at any time with <kbd>MOD</kbd>+<kbd>i</kbd>. (See
+[Shortcuts](#shortcuts) for what is meant by <kbd>MOD</kbd>).
```

Comment:
I'm not sure it's a good idea to explain on the options which happens to be the first: I think the options in the README is typically not read linearly, and options may be reordered in the future.

Reply:
Yeah, I know what you mean, and I wondered that myself. I just found it confusing reading several references to MOD before finding the Shortcuts section.

Maybe one solution could be to put the Shortcuts section nearer the top of the README, to increase the chance that it might be read before one of the other sections that reference MOD?

I'll remove the "(See Shortcuts...)" bit for now.